src/components/Profile.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
```

Comment:
Great job implementing another React Component and using props and router to be able to get to it! I think you've got a really good handle on how to use React and Router!